content/articles/london-nixos-meetup-1.md

```diff
@@ -0,0 +1,14 @@
+Title: First London NixOS meetup
+Date: 2015-03-26
+Short_summary: We are organising a NixOS meetup!
+Category: Events
+Authors: Vincent
+
+Hey everyone,  
+
+We are organising a NixOs meetup in London in [Mozilla offices](https://goo.gl/maps/EyM1r) the 21st of April at 7pm. 
+Tickets are limited and you can grab them on [Eventbrite](https://www.eventbrite.com/e/london-nixos-meetup-tickets-16331038576).  
+
+There will be pizzas, a talk and socializing.  
+Since this is the first event, we will try to figure out what people want out of it.
```

Comment:
Was just filling the article a bit so the article summary has something to cut x)